internal/controller/util/mw_util.go

```diff
@@ -700,6 +704,26 @@ func (mwu *MWUtil) DeleteManifestWork(mwName, mwNamespace string) error {
 	return nil
 }
 
+func (mwu *MWUtil) AddVRGToManifestWork(vrg *rmn.VolumeReplicationGroup, mw *ocmworkv1.ManifestWork) error {
```

Comment:
(nit) Can we change this function name from `AddVRGToManifestWork` to (say) `UpdateVRGManifest`? Initially I was thinking this is not calling the Update based on the name, and then looked at the change (which is correct). Hence the thought to rename this.

Reply:
Ack.